docs/resources/compensating-research-participants.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Compensating research participants
+parent: Resources
+nav_order: 6
+---
+
+# Compensating research participants
+{: .no_toc .fw-700 }
+
+How do we compensate folks for their time in our design research activities? There are plenty of reasons to do so…
+- [Compensating research participants @ 18F](https://ux-guide.18f.gov/research/plan/#compensating-research-participants)
+- [Equity centered design @ Digital.gov](https://digital.gov/files/equity-centered-design-julieben.pptx)
+- [Ethical and practical considerations of paying research participants](https://www.niehs.nih.gov/research/resources/assets/docs/ethical_and_practical_considerations_of_paying_research_participants_508.pdf)
+- [Why Design Researchers Should Compensate Participants](https://medium.com/notes-off-the-grid/why-design-researchers-should-compensate-participants-a65252352f67)
+- [The influence of risk and monetary payment](https://jme.bmj.com/content/30/3/293)
+- [Go beyond extractive "do no harm" to produce research that "heals"](https://twitter.com/albanvillamil/status/1311687313802645504)
+- [Qualitative Research at Code for America](https://info.codeforamerica.org/qualitative-research)
+
```

Comment:
There is a lot of content above, but it would probably be helpful to summarize a few of the main points (~5?) in a short bulleted list.

Reply:
added